### ISSUE TITLE
DeckGL + VueJS highlight color not displaying properly

### ISSUE THREAD

**Issue by FilipHusnjak** [visgl/deck.gl#4833]
#### Description
I'm using DeckGL  library with VueJS and ran into a possible bug. When i set property ```autohighlight``` to ```true``` inside ```SimpleMeshLayer``` inside ```mounted()``` method of VueJS lifecycle the highlight color is different from default one.
When i set the same layer with the exact same properties inside the method which is for example invoked when object is clicked or hovered, after click/hover event highlight color is correct. Its the same if i try to set highlightColor property myself, only when the click/hover event happens and method which creates layer again is invoked the proper highlight color is used. Until then different color is used (which is not even default one).

#### Repro Steps

```
mounted() {
  this.deck = new Deck({
      canvas: this.$refs.canvas,
      width: '100%',
      height: '100%',
      initialViewState: this.buildingViewState,
      layers: [
          new SimpleMeshLayer({
             id: 'test',
             coordinateSystem: COORDINATE_SYSTEM.METER_OFFSETS,
             data: [
             {
               position: [0, 0],
               color: this.buildingColor,
               orientation: this.buildingOrientation
             }
             ],
             mesh: url,
             pickable: true,
             autohighlight: true,
             highlightColor: [255, 0, 0, 128],
             onClick: info => this.click(info),
             getPosition: d => d.position,
             getColor: d => d.color,
             getOrientation: d => d.orientation,
             getScale: this.buildingScale
          })
        ]
    })
}

methods: {
   click (info) {
      this.deck.setProps({
          layers: [
             new SimpleMeshLayer({
                 id: 'test',
                 coordinateSystem: COORDINATE_SYSTEM.METER_OFFSETS,
                 data: [
                 {
                     position: [0, 0],
                     color: this.buildingColor,
                     orientation: this.buildingOrientation
                  }
                  ],
                  mesh: url,
                  pickable: true,
                  autohighlight: true,
                  highlightColor: [255, 0, 0, 128],
                  onClick: info => this.click(info),
                  getPosition: d => d.position,
                  getColor: d => d.color,
                  getOrientation: d => d.orientation,
                  getScale: this.buildingScale
              })
          ]
      })
   }
}

```
The code above does not show selected highlight color until the object is clicked.
The following highlight color is used before the click event:
![highlightColor](https://user-images.githubusercontent.com/41432967/88676390-482d2980-d0ec-11ea-9878-f0694fb83cff.png)

#### Environment (please complete the following information):
 - Framework Version: deck.gl 8.2.3
 - Browser Version: Chrome 84.0.4147.89
 - OS: Windows 10

#### Logs
There are no any warnings nor errors inside the console.

**Comment by Pessimistress** [comment]
@FilipHusnjak You should check:
1. If you are misspelling `highlightColor` in your code
2. If you are setting `layers` elsewhere without specifying `highlightColor`.

I do not use vue.js but I cannot think of any reason it would work differently with vue.

Here's an [example](https://codepen.io/Pessimistress/pen/wvMpqON) of it working. It will help us help you if you can reproduce the bug in Codepen.

**Comment by FilipHusnjak** [comment]
I found the problem. It works fine with ```new luma.CubeGeometry()``` as ```mesh``` property but the problem I have occurs when custom URL is provided and when loaderGL library is used.
I created demo VueJS app https://codesandbox.io/s/zen-darkness-vk0m2?file=/src/components/HelloWorld.vue .
You can see that highlight color is incorrect but when custom MESH_URL is replaced with luma.CubeGeometry() the color is correct (you have to adjust scaling for cubes).

**Comment by Pessimistress** [comment]
Fix published in 8.2.5
